### PR TITLE
Add lemon garlic chicken recipe

### PR DESCRIPTION
**Pull request by HugoKawamata**

### CODE REVIEW

File: recipes/lemonchicken.tex [hunk 1]
```diff
@@ -0,0 +1,27 @@
+\begin{recipe}{Lemon Garlic Chicken}{Serves 4}{2 hours}
+\Ing{1/3 cup olive oil}
```
Comment (MaxwellBo) [inline recipes/lemonchicken.tex]: should be `\fr{1}{3}`

File: recipes/lemonchicken.tex [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+\Ing{6 cloves of garlic}
+\Ing{1 large lemon}
+\Ing{1/2 tsp salt}
```
Comment (MaxwellBo) [inline recipes/lemonchicken.tex]: should be `\fr{1}{2}`

File: recipes/lemonchicken.tex [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+a glass or ceramic casserole dish. Pour the marinade over the chicken
+and use a spoon to redistribute the marinade until the chicken has been
+fully covered. Add the 4 remaining garlic cloves and sliced lemon. 
+Cover the dish tightly with foil and cook in oven for 90 minutes. 
+After 90 minutes, remove the foil, turn the oven up to 230C, and baste
+the chicken with the juices from the bottom of the dish (feel free to
+give the juice a little taste too, you can add more pepper or salt 
+to taste). Put the chicken back in for 20 minutes without foil
+to crisp up the skin.
+Serve with beans or other vegetables, with pan drippings spooned over.
```
Comment (MaxwellBo) [inline recipes/lemonchicken.tex]: Could probably put an `\Ing` describing how many beans

**Comment by TRManderson** [comment]
Please fix the things Max pointed out in another PR, merging for now

**Pull request merged**